src/main/java/com/google/sps/servlets/ColumnsFromTablesServlet.java

```diff
@@ -51,26 +49,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
       DatabaseClient dbClient = DatabaseConnector.getInstance().getDbClient(database);
 
-      String query = "";
-      String selectedTables = "";
-
-      //TODO: use StringBuilder rather than string concatenation
-
-      query = query + GET_COLUMNS_FROM_TABLES;
-      for (int i = 0; i < listOfTables.length; i++) {
-        //TODO: check if backslash is actually needed here
-        selectedTables = "\'" + listOfTables[i] + "\'";
-        query = query + selectedTables;
-        if (i != listOfTables.length-1) {
-          query = query + ", ";
-        } 
-      }
-      query = query + GROUP_BY_TABLE_NAMES;   
+      Statement query = QueryFactory.getInstance().buildColumnsQuery(listOfTables);
```

Comment:
very neat statement!

Reply:
^^ Definitely this is a neat way to reduce code here

Reply:
Thanks!